### ISSUE TITLE
Feature Request: Zoom in graph in web-interface

### ISSUE THREAD

**Issue by new-on-github** [Staacks/phyphox-webinterface#2]
Hopefully I don't miss something, but I cannot find a way to zoom the graph in the web-interface. Even if I zoom in the app, there is no zoomed/changed graph in the web-interface

This would be a really nice feature.

Btw: Thanks for this amazing piece of software!

**Comment by Staacks** [comment]
No, unfortunately, you did not miss anything. The problem is that the JavaScript graph library we use (chart.js) did not easily support what we needed when we implemented the zoom function and adding the zoom function would have required quite some work. Especially, as it requires an interface that works on touch devices as well as on traditional computers. So it went onto our todo list for later.

However, I am using the past tense here, because it has been a while since we looked into this problem and a quick search shows that chart.js is still under active development and has seen some major updates since then. So, I bump it up on our todo list for "re-evaluation" (although it might be a while before I even get to do that).
